app/controllers/ems_common.rb

```diff
@@ -767,10 +770,22 @@ def retrieve_vmware_cloud_api_versions
     [['vCloud API 5.1', '5.1'], ['vCloud API 5.5', '5.5'], ['vCloud API 5.6', '5.6'], ['vCloud API 9.0', '9.0']]
   end
 
-  def retrieve_openstack_security_protocols
+  def retrieve_nuage_api_versions
+    [['Version 3.2', 'v3_2'], ['Version 4.0', 'v4_0']]
```

Comment:
I'd like to see this come from the backend model.  Maybe the [Nuage `NetworkManager`](https://github.com/ManageIQ/manageiq/blob/50b0bed1db1461f0e5f78e9ccc7cfea70ace0625/app/models/manageiq/providers/nuage/network_manager.rb).